### PR TITLE
Install unzip into the image

### LINKED ISSUE #120: Unarchive missing from runners
##### ISSUE TYPE

 - Bug Report

##### SUMMARY
Unarchive module does not work within AWX as the program itself is missing from the runners

##### ENVIRONMENT
* AWX version: 21.0.0
* AWX install method: operator
* AWX deployment target: kubernetes
* Operating System: Ubuntu 20.04.4 LTS
* Web Browser: Any

##### STEPS TO REPRODUCE

Run any job that requires localhost to unarchive

##### EXPECTED RESULTS

File should be properly unarchived

##### ACTUAL RESULTS

The following error:

` "Failed to find handler for \"/home/runner/.ansible/tmp/ansible-tmp-1651699342.1886215-100-27628397993063/terraform_1.1.9_linux_amd64b1ssqluj.zip\". Make sure the required command to extract the file is installed. Unable to find required 'unzip' or 'zipinfo' binary in the path. Command \"/usr/bin/gtar\" could not handle archive." `

### PR DESCRIPTION
**Pull request by Himura2la**
Fix #120

### CODE REVIEW

**Comment by austlane** [comment]
Since this is necessary for a builtin to function. AWX-EE should probably have it included. 👍

File: _build/bindep.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@ git-lfs [platform:rpm]
 sshpass [platform:rpm]
 rsync [platform:rpm]
 epel-release [platform:rpm]
+unzip [platform:dpkg]
```
Comment (austlane) [inline _build/bindep.txt]:
```suggestion

```
This line is unnecessary, AWX-EE isn't a debian-based platform

AWX-EE will only match with these bindep platform profiles:
```shell
docker run --user="root" -it quay.io/ansible/awx-ee:latest bash
pip install bindep
bindep --profiles
```
```text
Platform profiles:
platform:base-py3
platform:centos
platform:centos-9
platform:redhat
platform:rpm
```

**Comment by Himura2la** [comment]
I was not sure, so I made it bulletproof. If debian-based systems are not required to support, I also propose https://github.com/ansible/awx-ee/pull/180, so that it does not confuse others (like it confused me).

**Comment by relrod** [comment]
@Himura2la Sorry for the delay here. I agree this probably should be included since an ansible-core module depends on it.

We've redone how the EE gets built now, could rebase your PR and instead add the line to the system requirements in `execution_environment.yml`?

Thanks much!

**Review by austlane — APPROVED** [review]
Looks good 👍

**Review by shanemcd — APPROVED** [review]

**Pull request merged**